src/internal/converters/eml/eml_test.go

```diff
@@ -0,0 +1,37 @@
+package eml
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/alcionai/corso/src/internal/tester"
+	"github.com/alcionai/corso/src/pkg/services/m365/api"
+)
+
+type EMLUnitSuite struct {
+	tester.Suite
+}
+
+func TestEMLUnitSuite(t *testing.T) {
+	suite.Run(t, &EMLUnitSuite{Suite: tester.NewUnitSuite(t)})
+}
+
+func (suite *EMLUnitSuite) TestConvert_messageble_to_eml() {
```

Comment:
I'd really love to see more tests around the variety of input data we could get. For example CC, BCC, senders that don't have a `Name` field filled out (just the email address), and the error paths in the code that's in `eml.go`. If you could please add tests for those it will help make sure things work like we think they do and that we don't accidentally break something going forward

Reply:
Fixed in https://github.com/alcionai/corso/pull/4651